dev/com.ibm.ws.session.cache_fat_infinispan/publish/servers/com.ibm.ws.session.cache.fat.infinispan.timeoutServerB/server.xml

```diff
@@ -17,11 +17,7 @@
         <feature>sessionCache-1.0</feature>
     </featureManager>   
     <httpSession maxInMemorySessionCount="1" allowOverflow="false" hideSessionValues="false" invalidationTimeout="5s" reaperPollInterval="30"/>
-    <httpSessionCache libraryRef="HazelcastLib" writeContents="ALL_SESSION_ATTRIBUTES">
-        <properties hazelcast.config.location="file:${shared.resource.dir}/hazelcast/${hazelcast.config.file}"/>
-    </httpSessionCache>
-    
-    <jndiEntry jndiName="jcache/hazelcast.config" value="${shared.resource.dir}/hazelcast/${hazelcast.config.file}"/>
+    <httpSessionCache libraryRef="InfinispanLib" writeContents="ALL_SESSION_ATTRIBUTES" uri="file:${shared.resource.dir}/infinispan/infinispan.xml"/>
```

Comment:
Couldn't find any documentation or usage of the properties supplied to cachingProvider.getCacheManager(uri, ..., properties) for Infinispan, and not any way to configure a URI in it, so it seems fine to replace this in the test.